dask_sql/context.py

```diff
@@ -801,7 +801,18 @@ def _get_ral(self, sql):
         schemas = self._prepare_schemas()
 
         # Now create a relational algebra from that
-        generator_builder = RelationalAlgebraGeneratorBuilder(self.schema_name)
+        ignore_case = self.schema[self.schema_name].config.get_config_by_prefix(
+            "dask.sql.identifier.ignore_case"
```

Comment:
I am following existing logic here using magic strings but think we should likely move all of the configuration keys to `mappings.py` and add further documentation in a subsequent PR. I am happy to make that PR as well once this one merges. Would prefer to not make that change in this PR since that change effects other codebase areas as well.